### DISCUSSION TITLE
Both supportedIds and excludedIds are not working.

### DISCUSSION THREAD

**Discussion by taowang1993** [danny-avila/LibreChat#3435] [Troubleshooting]
### What happened?

I am configuring the librechat.yaml file. 
All other settings are working, except supportedIds and excludedIds.
I tried the supportedIds first. It did not work. 
Then I tried the exlucedIds. But it did not work either. 
All my assistants are shown by click on the top right button on the right panel.

### Steps to Reproduce

 The file is using the following format.
supportedIds: ["asst_9Q4rsJPKiDPU2nwWjbWNwAc7"]

But the documentation is using thsi format.

supportedIds:
  - "asst_supportedAssistantId1"
  - "asst_supportedAssistantId2"

Which format should we use?

Thanks

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It does work, but not for admins. Check if your account you are testing with has the ADMIN role.

**Reply by danny-avila** [reply]
i.e. Admins can see all assistants, they are the first created user

**Comment by dportabella** [comment]
I had the same issue. I understand your answer. However this poses more questions, haha.
So, users have a role? which roles do exists? "admin" and (normal) "user"? how to see and configure which users are admin? I didn't see any configuration panel about user roles, nor any documentation.

**Reply by danny-avila** [reply]
I agree, we need more documentation on this, specifically for mentions of `supportedIds` and `excludedIds` and elsewhere, but we have a brief mention of this here:

> When you run the app for the first time, you need to create a new account by clicking on “Sign up” on the login page. The first account you make will be the admin account. The admin account doesn’t have any special features right now, but it might be useful for the admin dashboard to manage other users later.
> The first account created should ideally be a local account (email and password).

source: https://www.librechat.ai/docs/user_guides/authentication

The roles aren't used for much else at the moment, with the only current feature having explicit RBAC being Prompts, besides the one concerning this discussion.

This will be made more apparent when we update the documentation on Prompts, and the eventual release of the **Admin Panel.**

Relevant issue for tracking: https://github.com/danny-avila/LibreChat/issues/3137
